js/App.js

```diff
@@ -121,7 +120,7 @@ export class App extends Component {
       <View style={styles.container}>
         <StatusBar
           translucent={true}
-          backgroundColor="rgba(0, 0, 0, 0.2)"
+          backgroundColor="transparent"
```

Comment:
Can we keep a bit of background color so that if status-bar icons are over a white background (in our app) they are still visible?